app/styles/ui/_app-menu.scss

```diff
@@ -2,9 +2,11 @@
 
 #app-menu-foldout {
   display: flex;
+  max-height: 95%; // When at 100%, can cause some brief shifting of components
```

Comment:
The brief shifting of components would cause a `componentDidUpdate` to `setState` and could cause the app to crash. Plus I think it generally looks better to have a little bit of margin beneath the menu.